test/helpers/gas-usage.ts

```diff
@@ -40,24 +40,25 @@ export enum GasGroup {
 
   NETWORK_FEE_CHANGE,
   WITHDRAW_NETWORK_EARNINGS,
-  OPERATOR_FEE_INCREASE_LIMIT,
-  OPERATOR_DECLARE_FEE_LIMIT,
-  OPERATOR_EXECUTE_FEE_LIMIT,
+  DAO_UPDATE_OPERATOR_FEE_INCREASE_LIMIT,
+  DAO_UPDATE_DECLARE_OPERATOR_FEE_PERIOD,
+  DAO_UPDATE_EXECUTE_OPERATOR_FEE_PERIOD,
+  DAO_UPDATE_OPERATOR_MAX_FEE,
 
   CHANGE_LIQUIDATION_THRESHOLD_PERIOD,
   CHANGE_MINIMUM_COLLATERAL
 }
 
 const MAX_GAS_PER_GROUP: any = {
   /* REAL GAS LIMITS */
-  [GasGroup.REGISTER_OPERATOR]: 131890,
+  [GasGroup.REGISTER_OPERATOR]: 134500,
   [GasGroup.REMOVE_OPERATOR]: 70200,
   [GasGroup.REMOVE_OPERATOR_WITH_WITHDRAW]: 70200,
   [GasGroup.SET_OPERATOR_WHITELIST]: 84300,
   
   [GasGroup.DECLARE_OPERATOR_FEE]: 70000,
```

Comment:
@mtabasco please check what is the real number, cause this should have been increased

Reply:
@lior-blox It's a good number, around 100 extra gas for the extra check. No need to update.